client/components/ReviewAdd.jsx

```diff
@@ -0,0 +1,359 @@
+                    <label>
+                      <input name="Quality" type="radio" onChange = {() =>  this.handleQuality(4)}/>
+                      <span>Pretty great</span>
+                    </label>
+                    <label>
+                      <input name="Quality" type="radio" onChange = {() =>  this.handleQuality(5)}/>
+                      <span>Perfect</span>
+                    </label>
+                    </div>
+                  :null
+                }
+                {
+                  this.state.characteristics.Length?
+                    <div id='rate-length'>
+                    <label>
+                      <div>Length</div>
+                      <input name="Length" type="radio" onChange = {() =>  this.handleLength(1)}/>
+                      <span>Runs Short</span>
+                    </label>
+                    <label>
+                      <input name="Length" type="radio" onChange = {() =>  this.handleLength(2)}/>
+                      <span>Runs slightly short</span>
+                    </label>
+                    <label>
+                      <input name="Length" type="radio" onChange = {() =>  this.handleLength(3)}/>
+                      <span>Perfect</span>
+                    </label>
+                    <label>
+                      <input name="Length" type="radio" onChange = {() =>  this.handleLength(4)}/>
+                      <span>Runs slightly long</span>
+                    </label>
+                    <label>
+                      <input name="Length" type="radio" onChange = {() =>  this.handleLength(5)}/>
+                      <span>Runs long</span>
+                    </label>
+                    </div>
+                  :null
+                }
+                {
+                  this.state.characteristics.Fit?
+                    <div id='rate-fit'>
+                    <label>
+                      <div>Fit</div>
+                      <input name="Fit" type="radio" onChange = {() =>  this.handleFit(1)}/>
+                      <span>Runs tight</span>
+                    </label>
+                    <label>
+                      <input name="Fit" type="radio" onChange = {() =>  this.handleFit(2)}/>
+                      <span>Runs slightly tight</span>
+                    </label>
+                    <label>
+                      <input name="Fit" type="radio" onChange = {() =>  this.handleFit(3)}/>
+                      <span>Perfect</span>
+                    </label>
+                    <label>
+                      <input name="Fit" type="radio" onChange = {() =>  this.handleFit(4)}/>
+                      <span>Runs slightly long</span>
+                    </label>
+                    <label>
+                      <input name="Fit" type="radio" onChange = {() =>  this.handleFit(5)}/>
+                      <span>Runs long</span>
+                    </label>
+                    </div>
+                  :null
+                }
```

Comment:
May be able to use a function that will loop to save some code. Not sure how efficient it would be though

Reply:
> May be able to use a function that will loop to save some code. Not sure how efficient it would be though

will def consider!